File: app/src/main/java/com/cool/eye/mvp/java/test/TestActivity.java
package com.cool.eye.mvp.java.test;


import com.cool.eye.mvp.java.BaseMvpActivity;
import com.cool.eye.mvp.java.LoadView;

/**
 * @author ycb
 * @date 2018/8/17
 */
public class TestActivity extends BaseMvpActivity<TestPresenter> implements ITestView {


    @Override
    public LoadView getLoadView() {
        return LoadView.get(this);
    }

    @Override
    public TestPresenter getPresenter() {
        return new TestPresenter(this);
    }

    @Override
    public void setTest(String testMsg) {

    }

    @Override
    public void toAnotherPage() {

    }
}

File: app/src/main/java/com/cool/eye/mvp/kotlin/test/TestActivity.kt
package com.cool.eye.mvp.kotlin.test

import android.content.Context
import com.cool.eye.mvp.kotlin.BaseMvpActivity

/**
 * @author ycb
 * @date 2018/8/17
 */
class TestActivity : BaseMvpActivity<TestPresenter>(), ITestView {

    override fun getPresenter(): TestPresenter {
        return TestPresenter(this)
    }

    override fun setTest(testMsg: String) {

    }

    override fun withContext(): Context {
        return this
    }
}